### ISSUE TITLE
kick new project throws exception if bower is not installed globally 

### ISSUE THREAD

**Issue by codeinvain** [500tech/angular-kick#9]
here is the exception 
```
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn bower ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1046:32)
    at child_process.js:1137:20
    at process._tickCallback (node.js:355:11)
```
my workaround was `npm install -g bower`

**Comment by codeinvain** [comment]
same issue running kick server when gulp is not globally installed 
```
kick server
Running browserSync server on http://localhost:3001/
Press CTRL+C to stop
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn gulp ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1046:32)
    at child_process.js:1137:20
    at process._tickCallback (node.js:355:11)
    at Function.Module.runMain (module.js:503:11)
    at startup (node.js:129:16)
    at node.js:814:3
```
my workaround was `npm install -g gulp`

**Comment by ilyagelman** [comment]
Thank you! Fix published in 0.0.23
